### PR TITLE
Delete try catch in controllers

### PR DESCRIPTION
**Pull request by masteriza**

### CODE REVIEW

File: web/src/main/java/ua/com/vertex/dao/UserDaoImpl.java [hunk 1]
```diff
@@ -241,8 +241,8 @@ public int saveUserData(User user) {
                 "photo = :photo, " +
                 "discount = :discount, " +
                 "phone = :phone, " +
-                "role_id = (SELECT r.role_id FROM Roles r WHERE r.name= :role)" +
-                " WHERE user_id = :user_id";
+                "role_id = (SELECT r.role_id FROM Roles r WHERE r.name = :name) " +
```
Comment (backumich) [inline web/src/main/java/ua/com/vertex/dao/UserDaoImpl.java:244]: roleName?

File: web/src/main/java/ua/com/vertex/utils/Aes.java [hunk 2]
```diff
@@ -43,9 +43,8 @@ public static String encrypt(String value, String password) {
             cipher.init(Cipher.ENCRYPT_MODE, key);
             encrypted = cipher.doFinal(value.getBytes("UTF-8"));
         } catch (Exception e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException("Encrypt failed for - " + value, e);
```
Comment (backumich) [inline web/src/main/java/ua/com/vertex/utils/Aes.java]: encryption

File: web/src/main/java/ua/com/vertex/utils/Aes.java [hunk 3]
```diff
@@ -72,7 +71,7 @@ public static String decrypt(String value, String password) {
             byte[] decrypted = cipher.doFinal(encypted);
             toReturn = new String(decrypted, "UTF-8");
         } catch (Exception e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException("Decrypt failed for - " + value, e);
```
Comment (backumich) [inline web/src/main/java/ua/com/vertex/utils/Aes.java]: Decryption

**Review by backumich — CHANGES_REQUESTED** [review]

File: web/src/main/java/ua/com/vertex/dao/UserDaoImpl.java (hunk 1 above)
Comment (masteriza) [reply web/src/main/java/ua/com/vertex/dao/UserDaoImpl.java:244]: resolved

File: web/src/main/java/ua/com/vertex/utils/Aes.java (hunk 2 above)
Comment (masteriza) [reply web/src/main/java/ua/com/vertex/utils/Aes.java]: fixed

File: web/src/main/java/ua/com/vertex/utils/Aes.java (hunk 3 above)
Comment (masteriza) [reply web/src/main/java/ua/com/vertex/utils/Aes.java]: fixed

**Pull request merged**